processes/executive-copy-principles.md

```diff
@@ -1,39 +1,97 @@
+| Other numbers                            | When not a currency amount or in a mathematical/ technical context, numbers must be spelled alphabetically up to twelve inclusively.<br><br><li>_Three weeks at most; 6 weeks at most <li> Twelve components; 13 components_</li></br>|
+| Millions                                 | Numbers greater or equal to 1 million should be spelled out as long as the least non-zero decimal place is the ten thousand place.<br><br> <li>_1,000,000 = 1 million <li> 1,500,000 = 1,5 million <li> 1,550,000 = 1,55 million <li> 1,555,000 = 1,555,000_</li></br>|
 
-As the entity responsible for producing the executive copy, you must ensure that readability and understanding of the vote do not require in-depth knowledge of the technical aspects of the Maker Protocol. It is important that the level of technical detail in the executive copy be carefully considered and managed. This ties into all of the above principles. In most cases, the technical detail of how an outcome is achieved obscures clarity and clutters the copy with information that is not a strict priority.
+|                     | Parameters |
+|:-------------------:| -----------|
+| New parameters      | When the parameter is established for the first time (e.g., when onboarding a new module or function), it must be quoted as:<br> `$parameter_name will be set to $value`</br><br><li>_Maximum Debt Ceiling will be set to 50 million DAI.<li>Ceiling Increase Cooldown will be set to 12 hours._</li></br> |
+| Existing parameters | When the proposal refers to the modification of an existing parameter, the change must be quoted as:<br>`$parameter_name will be $change by $change_value from $original_value to **$new_value**`<br> The new value must be marked in **bold**.<br> In a set of parameters, all values should be listed, even if they remain unchanged.<br><br> <li>_Target Available Debt will be decreased by 20 million from 400 million to **380 million**. <li> DC-IAM line (max DC) remains unchanged at 10B._</li></br>| 
+|Percentage change    |  Differences in numbers expressed as percentages must be fully spelled out.<br><br><li>_The DSR will decrease by 1 percentage point, from 8% to 7%._ </li><br> The percentage difference between two non-percentual numbers must be quoted as the `%` value.<br><li>_The supply of DAI will increase by 20% from 100,000 DAI to 80,000 DAI._</li><br> Note that typically deltas are specified as nominal and not percentual values, so this would be applicable only in exceptional cases.</br>|
```

Comment:
Tried to improve the wording.

Your example is a decrease not an increase 🙂 

```suggestion
|Percentage change    |  Differences in numbers expressed as percentages must be fully spelled out. Care should be taken to differentiate percentage points and percentage change.<br><br><li>_The DSR will decrease by 1 percentage point, from 8% to 7%._ </li><br> The percentage difference between two non-percentual numbers must be quoted as the `%` value.<br><li>_The supply of DAI will decrease by 20% from 100,000 DAI to 80,000 DAI._</li><br> Note that typically deltas are specified as nominal and not percentual values, so this would be applicable only in exceptional cases.</br>|
```